src/Files.App/Services/NetworkDrivesService.cs

```diff
@@ -188,13 +203,13 @@ public async Task<bool> AuthenticateNetworkShare(string path)
 				}
 			}
 
-			if (connectionError == Win32Error.NO_ERROR)
+			if (res == WIN32_ERROR.NO_ERROR)
 			{
 				return true;
 			}
 			else
 			{
-				await DialogDisplayHelper.ShowDialogAsync("NetworkFolderErrorDialogTitle".GetLocalizedResource(), connectionError.ToString().Split(":")[1].Trim());
+				await DialogDisplayHelper.ShowDialogAsync("NetworkFolderErrorDialogTitle".GetLocalizedResource(), res.ToString());
```

Comment:
Any reason to remove the ".Split(":")[1].Trim()"?

Reply:
To string in Vanara does convert to friendly error text right?

Reply:
Yes, it does.
In the meantime, we can leave as it while am we also cloud use FormatMessage.

Reply:
Found out in the first place this is not needed.

https://github.com/files-community/Files/blob/ea1818541cebe09d7aa33cb3a9d86857c82e0607/src/Files.App/Strings/en-US/Resources.resw#L2217-L2219